client/sections.js

```diff
@@ -222,6 +222,14 @@ if ( config.isEnabled( 'reader' ) ) {
 	} );
 
 	sections.push( {
+		name: 'reader-post-recomendations',
+		paths: [ '/recommendations' ],
```

Comment:
Good call. I've made this more specific on the path (/recommendations/posts) and placed above the other recs. The original Recommendations gets loaded correctly in both scenarios (when /posts is above and below the original /recommendations).